### DISCUSSION TITLE
Draco Loader renders nothing

### DISCUSSION THREAD

**Discussion by sunyonglincanada** [visgl/deck.gl#5289] [Q&A] [answered]
Hi,

I was trying to load one compressed gltf file using Draco loader according to the doc - https://loaders.gl/modules/gltf/docs/api-reference/gltf-loader but got nothing to render. Here is the example - https://codepen.io/sunyonglincanada/pen/MWjOOJZ?editors=1111. 

Can anyone help me figure this out? Thanks in advance!!

**Comment by Pessimistress** [comment, marked as answer]
@ibgreen An error is thrown during `loadImages` because it's trying to access `gltf.buffers` before it's populated. https://github.com/visgl/loaders.gl/blob/master/modules/gltf/src/lib/parse-gltf.js#L27-L35

@sunyonglincanada It would be more helpful next time if you report your console errors.

**Reply by ibgreen** [reply]
https://github.com/visgl/loaders.gl/issues/1073

**Reply by sunyonglincanada** [reply]
Thanks @Pessimistress and @ibgreen for the updates and will do next time for sure.

**Comment by sunyonglincanada** [comment]
Hi @ibgreen and @Pessimistress  -  I've found the https://github.com/visgl/loaders.gl/issues/1073 is closed and merged into master. Then I updated the loaders.gl version to v2.3.8 and try the Draco loader but did not render anything with one error in the console - https://imgur.com/a/24srODB. Not sure if I updated the correct version. Can you please take a look?  Thanks in advance:)

**Reply by Pessimistress** [reply]
Please ask in the loaders.gl repo what’s included in 2.3.8.

**Reply by ibgreen** [reply]
Apologies, the patch publishing in loaders.gl has fallen behind we will try to get a 2.3.9 out soon.

**Reply by sunyonglincanada** [reply]
Thanks @Pessimistress and @ibgreen for the updates!! I will try it once v2.3.9 is released.

**Reply by dryabinin94** [reply]
@sunyonglincanada Good day! v2.3.9 is released. Please check the issue is gone and let us know if something is wrong.

**Reply by sunyonglincanada** [reply]
> @sunyonglincanada Good day! v2.3.9 is released. Please check the issue is gone and let us know if something is wrong.

Gotcha. Thanks @dryabinin-actionengine for letting me know.  I will update the version to 2.3.9 and have a try.
